cmd/slinky-config/main.go

```diff
@@ -302,7 +315,7 @@ func init() {
 		&updateInterval,
 		"update-interval",
 		"",
-		500*time.Millisecond,
+		250*time.Millisecond,
```

Comment:
This is a relatively small change, but in general I think we should avoid combining unrelated changes into PRs. It makes it more likely for people to miss them.  
Updating the default price aggregation interval has nothing to do with adding uniswap markets.

Reply:
yea agreed, this was meant to be a fix in previous PRs but got excluded.